assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -1000,9 +1000,11 @@ def __init__(self, n_retry, backoff_factor):
 
         retry_strategy = Retry(
             total=n_retry,
```

Comment:
total and status with the same value?

Reply:
<img width="634" alt="image" src="https://github.com/Azure/azureml-assets/assets/42190563/d88963ee-99cc-42bd-81da-229fa4f0dd9f">
<img width="580" alt="image" src="https://github.com/Azure/azureml-assets/assets/42190563/396a00bd-474a-48e0-b2f2-d95023539d9c">

do you think we should set total to None ?

Reply:
I see, I was interpreting the meaning from the parameter name :)

Reply:
yeah it doesn't seem to make a lot of sense imo